### ISSUE TITLE
Medusa not restarting on Qnap

### ISSUE THREAD

**Issue by papampi** [pymedusa/Medusa#9888]
Medusa Info: | Branch: master Commit: 21e350cd4a21f6fb65f750ab5c2d1e9c92c621d6 
-- | --
Version: |0.5.18 
Database: |44.18
Python Version: | 3.9.6 (default, Aug 22 2021, 06:41:01) [GCC 8.4.0]
SSL Version: | OpenSSL 1.1.1k 25 Mar 2021
OS: | Linux-4.2.8-x86_64-with-glibc2.27
Locale: | en_US.UTF-8

Since a while after auto update or manually clicking restart its not restarting and I should manually restart it from app center or command line.
When I manually tried to run the start up command after clicking restart I got this:

```
[~] # /opt/bin/python3 /share/CACHEDEV1_DATA/.qpkg/OMedusa/OMedusa/start.py --daemon --nolaunch --datadir /share/CACHEDEV1_DATA/.qpkg/OMedusa/config --config /share/CACHEDEV1_DATA/.qpkg/OMedusa/config/config.ini --pidfile /var/run/OMedusa.pid
PID file: /var/run/OMedusa.pid already exists. Exiting.
```
So removed the pid file and run the command and it started. 
It looks like its not removing the pid file before closing Medusa to run the restart command

**Last Lines of log :**
```
2021-09-18 20:16:41 INFO     EVENT-QUEUE :: [d4ffa66] Aborting all threads
2021-09-18 20:16:41 INFO     EVENT-QUEUE :: [d4ffa66] Restarting Medusa with ['/opt/bin/python3', '/share/CACHEDEV1_DATA/.qpkg/OMedusa/OMedusa/start.py', '--daemon', '--nolaunch', '--datadir', '/share/CACHEDEV1_DATA/.qpkg/OMedusa/config', '--config', '/share/CACHEDEV1_DATA/.qpkg/OMedusa/config/config.ini', '--pidfile', '/var/run/OMedusa.pid']
```

**Comment by medariox** [comment]
Please report this to sherpa.

**Comment by papampi** [comment]
@medariox 
No Fix for #9899 ?
